frontend/util.lua

```diff
@@ -127,18 +127,19 @@ function util.secondsToClock(seconds, withoutSeconds)
         end
     else
         local round = withoutSeconds and require("optmath").round or passthrough
-        local hours = string.format("%02d", seconds / 3600)
+        local days = string.format("%02d", seconds / (24*3600)) --implicit math.floor for string.format
+        local hours = string.format("%02d", (seconds / 3600) % 24)
         local mins = string.format("%02d", round(seconds % 3600 / 60))
         if withoutSeconds then
             if mins == "60" then
                 -- Can only happen because of rounding, which only happens if withoutSeconds...
                 mins = string.format("%02d", 0)
                 hours = string.format("%02d", hours + 1)
             end
-            return hours .. ":" .. mins
+            return  (days ~= "00" and (days .."d") or "") .. hours .. ":" .. mins
         else
             local secs = string.format("%02d", seconds % 60)
-            return hours .. ":" .. mins .. ":" .. secs
+            return (days ~= "00" and (days .. "d") or "") .. hours .. ":" .. mins .. ":" .. secs
```

Comment:
Dunno where secondsToClock() is used, but I see it is used in the statistics plugin.
Will we always get a string in day when hours > 24 or does that depends on some other option ?
In my current book, Current statistics show now:
```
Estimated time to read: 32:21:13.
Estimated reading finished: 2022-01-06 (42 days)
```
In this context, I prefer seeing `32:21:13` (which means 32 hours of active reading) to `1d:6:21:13` which talks to me less than "32 hours" (I'll be doing 24+6 = 32 in my head).
So, does this changes affects this one? If yes, does it need some additional option to prevent this ?